docs/source/api/v2/steering.rst

```diff
@@ -0,0 +1,97 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-steering:
+
+************
+``steering``
+************
+
+``GET``
+=======
+Gets a list of all :ref:`Steering Targets <steering-qht>` in the Traffic Ops database.
+
+:Auth. Required: Yes
+:Roles Required: "Portal", "Steering", "Federation", "operations" or "admin"
+:Response Type:  Array
+
+Request Structure
+-----------------
+No parameters available.
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/2.0/steering HTTP/1.1
+	User-Agent: python-requests/2.22.0
+	Accept-Encoding: gzip, deflate
+	Accept: */*
+	Connection: keep-alive
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+:deliveryService:	A string that is the :ref:`ds-xmlid` of the steering :term:`Delivery Service`
+:clientSteering:	Whether this is a :ref:`client steering <ds-client-steering>` delivery service.
+:targets:			The delivery services that the steering delivery service targets.
+:filters:			Filters of type :ref:`STEERING_REGEXP <ds-steering-regexp>` that exist on either of the targets.
```

Comment:
Sub-property definitions?

Reply:
Sub-property definitions added in dd903c8f01.